fastlane/Fastfile

```diff
@@ -628,50 +704,48 @@ def submit_to_store
   end
 end
 
-def upload_to_s3
-  unless ENV['AWS_BUCKET_NAME'].nil? || ENV['AWS_BUCKET_NAME'].empty? || ENV['AWS_REGION'].nil? || ENV['AWS_REGION'].empty?
-    apk_path = lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
-    ipa_path = lane_context[SharedValues::IPA_OUTPUT_PATH]
-    s3_region = ENV['AWS_REGION']
-    s3_folder = "#{ENV['AWS_FOLDER_NAME']}/#{ENV['BRANCH_TO_BUILD']}"
-    s3_bucket = ENV['AWS_BUCKET_NAME']
-    s3 = Aws::S3::Resource.new(region: s3_region)
-    abbreviated_commit_hash = last_git_commit[:abbreviated_commit_hash]
-    filename = ENV['BRANCH_TO_BUILD']
+def post_pr_message(options)
```

Comment:
we should post to github only once with the links to both files.

Reply:
Hmm, this is me picking a wrong name.  I'll update the name.
This function is posting a message into the Mattermost instance as webhook and not as a comment into Github.
For Mattermost webhook, I can see that we have two posts - one for Android and another for iOS which were not changed.
For Github comment, it's handled separately -> [BuildMobileAppDoneMessage](https://github.com/mattermost/platform-private/blob/master/mattermod/config-mattermod.json#L116) that's triggered by [mattermod](https://github.com/mattermost/mattermost-mattermod/blob/65a2a912cfd0255818cc51dede8ae3303d84eb1c/server/spinmint.go#L271) on success.

Reply:
Can't think of a better name.  I've change it to `qa_build_message`